pkg/plugin/sdk/v1alpha2/plugin/interface.go

```diff
@@ -27,3 +27,30 @@ type Plugin interface {
 		promptRunner ioutils.PromptRunner,
 	) error
 }
+
+// Base satisfies the Plugin interface. For plugins that only implement a subset
+// of the Plugin interface, using this as a base will give the advantage of not
+// needing to implement the empty methods.
+type Base struct{}
```

Comment:
Since this is public the name is essentially final (without a breaking change) so we should think about it...

Will this always be accessed with the package name such as `plugin.Base`? Or should the struct itself have more then just "base"

Reply:
I thought about naming it also `Empty`, but it doesn't read well when consumed by the real plugin implementation.
Yes to the first question. No to the second question.

I'm happy to run alternative names with other folks. This is not as _final_ as you may think. This is SDK v1alpha2. Ideally, we wouldn't change but nothing will forbid us from doing so.